config/Abigail.am

```diff
@@ -15,6 +15,11 @@
 PHONY += checkabi storeabi
 
 checkabi:
+	if test "$$(getconf LONG_BIT)" -ne 64; then \
+		echo '*** ABI definitions provided apply only to 64-bit architectures'; \
+		echo '*** Skipping `checkabi` target and assuming success.'; \
+		exit 0; \
```

Comment:
I propose to set exit code to 1, we already have some checks and return 1 there https://github.com/openzfs/zfs/blob/master/Makefile.am#L148

Reply:
Fair enough. I went about it little bit differently: moving abigail-related checks to `config/Abigail.am`.